src/platform/graphics/texture.js

```diff
@@ -186,7 +186,7 @@ class Texture {
      * - {@link FUNC_NOTEQUAL}
      *
      * Defaults to {@link FUNC_LESS}.
-     * @param {Uint8Array[]|HTMLCanvasElement[]|HTMLImageElement[]|HTMLVideoElement[]|Uint8Array[][]} [options.levels]
+     * @param {Uint8Array[]|Uint16Array[]|Uint32Array[]|Float32Array[]|HTMLCanvasElement[]|HTMLImageElement[]|HTMLVideoElement[]|Uint8Array[][]} [options.levels]
```

Comment:
We could just use `ArrayBufferView` which represents any typed array. This is what mesh API for vertex data uses:
`     * @param {number[]|ArrayBufferView} data - Vertex data for the specified semantic.`